### ISSUE TITLE
Bourbon 5.0.0 does not work with sassc-rails

### ISSUE THREAD

**Issue by jhawthorn** [thoughtbot/bourbon#1047]
## Description

When upgrading bourbon on a Rails 5.1 project with `sassc-rails` I encountered the error:

```
failing on `Error: File to import not found or unreadable: bourbon.
```

I would guess this is an issue since 54b73b5b04b4c3485492aedee0f1c4d8daac3fb5 (but haven't confirmed)

## Steps to Reproduce

1. Generate a new rails project.
1. Add sassc-rails and bourbon to the Gemfile.
1. Create an `application.scss` with `@import 'bourbon'`
1. Try to load a page in development (or `rake assets:precompile`)
1. :bomb:

**Comment by tysongach** [comment]
Hi @jhawthorn! Thanks for reporting this.

I'm able to reproduce the issue. I fiddled with our load paths, even reverting https://github.com/thoughtbot/bourbon/commit/54b73b5b04b4c3485492aedee0f1c4d8daac3fb5, but didn't appear to fix it.

We'll have to do some more digging.

**Comment by sheelah** [comment]
I'm getting this same error in a JavaScript project, using `node-sass-chokidar` to build Sass:

```
> node-sass-chokidar --include-path=node_modules/bourbon/app/assets/stylesheets --include-path=node_modules/bourbon-neat/app/assets/stylesheets src/scss -o src/scss

{
  "status": 1,
  "file": "/Users/sheelah/src/myproject/src/scss/patterns/_base.scss",
  "line": 9,
  "column": 1,
  "message": "File to import not found or unreadable: bourbon.\nParent style sheet: /Users/sheelah/src/myproject/src/scss/patterns/_base.scss",
  "formatted": "Error: File to import not found or unreadable: bourbon.\n       Parent style sheet: /Users/sheelah/src/myproject/src/scss/patterns/_base.scss\n        on line 9 of src/scss/patterns/_base.scss\n>> @import 'bourbon';\n   ^\n"
}
```

**Comment by tysongach** [comment]
@sheelah Given that this issue is rooted in an issue with `sassc-rails`, would you mind opening a new issue for that? The JavaScript environment is likely a different problem and has a different solution than this one.

Thanks!

**Comment by sheelah** [comment]
@tysongach Sure thing.

**Comment by botandrose** [comment]
Downgrading to Bourbon 4.3.4 fixed this for me, so the regression is inbetween that release and 5.0.0. I'm going to try to run a bisect.

**Comment by botandrose** [comment]
73b09f618bd3189bf5af6f639e3df07680ed4fc7 is the culprit

**Comment by botandrose** [comment]
Partially reverting the offending commit yields the following `lib/bourbon.rb`, and fixes the issue. Opening a PR.

```ruby
require "sass"
require "bourbon/generator"

module Bourbon
  if defined?(Rails) && defined?(Rails::Engine)
    class Engine < ::Rails::Engine
      config.assets.paths << File.expand_path("../../core", __FILE__)
    end
  else
    Sass.load_paths << File.expand_path("../../core", __FILE__)
  end
end
```

**Comment by guilleiguaran** [comment]
That is more important now that ruby sass is being deprecated and Rails will start shipping `sassc-rails` by default: https://github.com/sass/sass/issues/2483

**Comment by robertmain** [comment]
It's because bourbon keep changing the repo layout every couple of versions and it's driving me crazy! What's up with that?

**Comment by tysongach** [comment]
We just released Bourbon v5.0.1, which should fix this issue.
